src/vl-functional-header.js

```diff
@@ -0,0 +1,95 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/vl-ui-link/dist/vl-link.js';
+import '/node_modules/vl-ui-icon/dist/vl-icon.js';
+
+/**
+ * VlFunctionalHeader
+ * @class
+ * @classdesc Toont bovenaan de pagina generieke informatie zonder af te leiden zoals bijvoorgeeld titel, acties, tab navigatie of zoek input.
+ * 
+ * @property {String} data-vl-link - Attribuut wordt gebruikt om de link van de titel te bepalen.
+ * @property {String} data-vl-title - Attribuut wordt gebruikt om de tekst van de titel te bepalen.
+ * @property {String} data-vl-sub-title - Attribuut wordt gebruikt om de tekst van de sub titel te bepalen.
+ * 
+ * @extends VlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-functional-header/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-functional-header/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-functional-header.html|Demo}
+ * 
+ */
+export class VlFunctionalHeader extends VlElement(HTMLElement) {
+    static get _observedAttributes() {
+        return ['title', 'sub-title', 'link'];
+    }
+
+    constructor() {
+        super(`
+            <style>
+                @import '/src/style.css';
+                @import '/node_modules/vl-ui-link/dist/style.css';
+                @import '/node_modules/vl-ui-icon/dist/style.css';
+            </style>
+            <header class="vl-functional-header" role="banner">
+                <div class="vl-layout">
+                    <div class="vl-functional-header__row">
+                        <div class="vl-functional-header__content">
+                            <h1 class="vl-title">
+                                <a id="title" class="vl-functional-header__title">
```

Comment:
deze "is" dan niet vl-link?

Reply:
Nee, aangezien vl-link sowieso `vl-link` als class toevoegt en dat hier blijkbaar niet gewenst is.